lti_emailer/settings/local.py

```diff
@@ -15,3 +15,14 @@
 }
 
 dictConfig(LOGGING)
+
+CANVAS_URL = 'http://canvas.icommons.harvard.edu/'
```

Comment:
Thanks, and done. Removed the CANVAS_URL references and added to secure settings instead.  
As for general URL values, I had discussed briefly with Josie. I could also move the emailer_tool_relative_url to secure settings, but the test is dependent on the site setup so the relative url should not be modified.  Before I make another change (I am curious), do URLs really belong to secure_settings? (if they aren't secure settings like username/password?)  What are the general best practices?